### PR TITLE
Ansible scripts

### PR DESCRIPTION
**Pull request by lcoram**

### CODE REVIEW

**Comment by lcoram** [comment]
closes https://github.com/metno/lard/issues/9

File: ansible/create-project-network.yml
Comment (intarga) [inline ansible/create-project-network.yml:1]: Do these whole files need to be vaulted? Is it not possible to create a secrets file that they use, so the structure can stay open? @lcoram

Comment (lcoram) [reply ansible/create-project-network.yml:1]: Since the restructure now only the vars file for that particular role is vaulted

**Comment by lcoram** [comment]
Now there are just 2 steps, provision and configure (arnulf mentioned that it was better to seperate those two parts). Also, a way to do the switchover.

**Comment by lcoram** [comment]
Rebased to fix conflicts with changes in db schema

**Comment by intarga** [comment]
> We have tested that Manuel can run this, probably ready to be merged

Yes, sorry I just forgot to do this

**Review by intarga — APPROVED** [review]

**Pull request merged**